src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -15,11 +15,11 @@ import FastImage from 'react-native-fast-image';
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
 import MemoIcArrowBackCircle from '../../../assets/arrow/Ic_arrow_back_circle';
 import ShareIconCircle from '../../../assets/icons/Ic_share_circle';
-import TopicDefaultIcon from '../../../assets/topic.png';
+import TopicSignedDefaultIcon from '../../../assets/dp-topics-signed.png';
+import TopicAnonDefaultIcon from '../../../assets/dp-topics-anon.png';
 import {Shimmer} from '../../../components/Shimmer/Shimmer';
 import dimen from '../../../utils/dimen';
 import {normalize} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `TopicSignedDefaultIcon` and `TopicAnonDefaultIcon` is a good change as it allows for different icons based on the follow type. However, please ensure that these assets are correctly added to the project and their paths are correct.



<!-- This is an auto-generated comment by OSS CodeRabbit -->